src/pages/saksbehandling/attestering/sharedStyles.module.less

```diff
@@ -1,11 +1,19 @@
 @import '~/styles/variables.less';
 
-.container {
-    width: 100%;
+//Dette er ikke en 'shared' styling. den ligger her så jeg ikke trenger å lage en helt ny css fil for Attestering.tsx ;)
```

Comment:
Kremt ;) @simejo Kanskje vi skal flytte all styling til en fil for enkelhetens skyld :p

Reply:
😄

Reply:
Ville nå lagd den ekstra klassen ;) Gjør det mye enklere å vedlikeholde senere. Eksempelvis hvis man skal slette komponenten, så kan man bare slette tilhørende styling-fil i samme slengen